sanitizers/BUILD.bazel

```diff
@@ -1,6 +1,8 @@
 java_library(
     name = "sanitizers",
-    visibility = ["//src/main/java/com/code_intelligence/jazzer/runtime:__pkg__"],
+    visibility = [
+        "//visibility:public",
```

Comment:
Some of the sanitizers need to be on the bootstrap classpath as they have to be visible to e.g. `java.util.regex.Pattern`. The `runtime` package is bundled into a JAR that is appended to the boostrap classpath at runtime. This visibility declaration ensures that we don't accidentally mess up this separation, so I would like to preserve it. 

Is there a way to add classes to the bootstrap classpath on Android? I would expect it to be necessary as otherwise Jazzer may crash when e.g. `Pattern#compile` is called.

Reply:
Ack, I will look into this more.

Reply:
I have tested all the example sanitizers, and they all either catch the security error or they run forever, but none crash due to not being able to find the Code Intelligence class from the runtime package.

OsCommandInjectionProcessBuilder
OsCommandInjectionRuntimeExec
RegexRoadblocks
StackOverflowRegexInjection 

These examples all run forever. Is this possibly because we are instrumenting offline, so Pattern#compile never gets instrumented so it never catches the security issue, but also doesn't crash or need these classes loaded to the bootstrap classpath?

Most of the sanitization examples work however, and this was also expected that some sanitizers wouldn't work if we did offline sanitization.

This being said, is it still important to find a way to load these classes into the bootstrap classpath? Or would an acceptable alternative be to create another bazel build target like this:

java_library (
  name = "offline_sanitizers",
  <srcs = sanitizers needed for offline running of Jazzer>
  ...
)

Reply:
It's fine not to load them. 

Could you create a target next to this one called `offline_only_sanitizers` that has the required visibility and use it for this purpose? You don't need to filter out the sanitizers that don't work just yet.